cpp/benchmarks/join/join_common.hpp

```diff
@@ -193,3 +204,47 @@ void BM_join(state_type& state, Join JoinFunc, int multiplicity = 1, double sele
     set_throughputs(state);
   }
 }
+
+template <bool Nullable,
+          join_t join_type                  = join_t::HASH,
+          cudf::null_equality compare_nulls = cudf::null_equality::UNEQUAL,
+          typename state_type,
+          typename Join>
```

Comment:
Not necessarily a change request, but we might eventually want to consider passing some of the parameters differently, e.g. using strings instead of template parameters, e.g. 
https://github.com/rapidsai/cudf/blob/da6ce2a96b616891d66c4f41ade9db4047eb4b3f/cpp/benchmarks/string/find.cpp#L34-L35
 As the latter could lead to significantly higher compile times.